<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSelectCameraIndexClass</class>
 <widget class="QDialog" name="DlgSelectCameraIndexClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DlgSelectCameraIndex</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,3,2" rowminimumheight="0,3,2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>选择要设置的相机：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSpinBox" name="spBox_cameraIndex">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbtn_ok">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
